src/settings.cpp

```diff
@@ -1585,20 +1668,32 @@ static IniFile *IniLoadConfig()
  */
 void LoadFromConfig(bool startup)
 {
-	IniFile *ini = IniLoadConfig();
+	IniFile *generic_ini = IniLoadConfig();
+	IniFile *private_ini = new IniFile();
```

Comment:
Can't you just put them on the stack, instead of `new` + `delete`? Would mean some `&` in line 1685 though.